mne_bids/path.py

```diff
@@ -142,7 +142,7 @@ def _find_matched_empty_room(bids_path):
 
 
 class BIDSPath(object):
-    """A BIDS path object.
+    """A BIDSPath object.
```

Comment:
I don't agree with any of these changes from "BIDS path" to "BIDSPath". The idea of the original doctrings was to convey that we're talking about paths in the BIDS dataset. Now you've turned all of those into self-referential thingies, which I think isn't helpful. For example here, you explain a BIDSPath b saying that it's a BIDSPath, this is … circular.

I do agree that the docstring should get reworked, but I don't think these changes here are the way we should go.

I hope this doesn't sound too harsh :D

Reply:
just for clarification, my comment above only applies to the changes in the docstring(s), not to the inline comments in the code or tests